app/controllers/users_controller.rb

```diff
@@ -1,9 +1,22 @@
 class UsersController < ApplicationController
   def index
+    @users = Users.all
     puts params
   end
 
```

Comment:
Please add `bundle install`, `rails db:create`, `rails db:migrate`, and `yarn or npm install` in README as these commands are necessary to install the project locally.